pkg/volume/flocker/flocker.go

```diff
@@ -421,6 +421,10 @@ func (c *flockerVolumeUnmounter) TearDown() error {
 
 // TearDownAt unmounts the bind mount
 func (c *flockerVolumeUnmounter) TearDownAt(dir string) error {
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
+		glog.Infof("The dir %v doesn't exist as it might have been unmounted by a previous unmount operation", dir)
+		return nil
+	}
 	notMnt, err := c.mounter.IsLikelyNotMountPoint(dir)
```

Comment:
If we see the code of `IsLikelyNotMountPoint` it looks like:

```
	// Check the directory exists
	if _, err = os.Stat(file); os.IsNotExist(err) {
		glog.V(5).Infof("findmnt: directory %s does not exist", file)
		return true, err
	}
``` 

So basically same check as being done here. Does it make sense to handle the returned error from the `IsLikelyNotMountPoint` check and log and return `nil` rather than doing this thing twice?